packages/gatsby-source-wordpress/src/normalize.js

```diff
@@ -138,11 +138,15 @@ exports.liftRenderedField = entities =>
 
 // Exclude entities of unknown shape
```

Comment:
Can you add some context to function description comment so it's easier for future contributors? That we assume that all entities need ids except for whitelisted types (currently just settings)